autonomy/deploy/generators/docker_compose/base.py

```diff
@@ -39,12 +43,17 @@
 )
 from autonomy.deploy.generators.docker_compose.templates import (
     ABCI_NODE_TEMPLATE,
+    ACN_NODE_TEMPLATE,
     DOCKER_COMPOSE_TEMPLATE,
+    HARDHAT_NODE_TEMPLATE,
     TENDERMINT_CONFIG_TEMPLATE,
     TENDERMINT_NODE_TEMPLATE,
 )
 
 
+N_RESERVED_IP_ADDRESSES = 4
```

Comment:
We will add one ACN and hardhat nodes at the top with IP being `192.167.11.2` and `192.167.11.3` so we will start from `192.167.11.4` for abci and tendermint nodes